api/src/main/java/com/capitalone/dashboard/rest/DeployController.java

```diff
@@ -50,4 +62,23 @@ public DeployController(DeployService deployService) {
                 .status(HttpStatus.CREATED)
                 .body(response);
     }
+    
+    @RequestMapping(value = "/deploy/rundeck", method = POST,
+            consumes = TEXT_XML_VALUE, produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> createRundeckBuild(HttpServletRequest request,
```

Comment:
@jmcshane  - you can a create a GenericXMLparser and make RundeckXMLParser extend GenericXMLParser. In that way code will be more reusable.Yes, using Jackson streaming api to serialize XML to JSON may be complicated when xml metadata is unknown. Question - Will rundeck produces various XML output depending on how it is configured?